pennylane/data/__init__.py

```diff
@@ -12,7 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-The data module provides functionality to load and manage datasets.
+The data subpackage provides functionality to access, store and manipulate quantum datasets within the PennyLane framework.
```

Comment:
we say "within the PennyLane framework" a few times here. is that a normal way to refer to this, a framework? Why not just "with PennyLane"?

Reply:
Nice, actually the end of this sentence is unnecessary. We can just say:
"the data subpackage provides functionality to access, store and manipulate quantum datasets"